src/Locale.php

```diff
@@ -85,9 +88,11 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
         $this->urlGenerator->setDefaultArgument($this->queryParameterName, $locale);
 
         if ($request->getMethod() === Method::GET) {
+            $location = rtrim($this->aliases->get($this->baseUrlAlias), '/') . '/'
```

Comment:
Could be extracted to a separate method and reused. The only difference is `$locale . $path` / `$newPath`, which can be handled via parameter. And response part is identical too.

Reply:
This is quite simple piece of code. I'm not sure it worth a method.